README.md

```diff
@@ -25,7 +25,7 @@ The main purpose (and test case) to build it was to allow streaming from [**OBS
 * For the simplest case, just run a container with this image:
 
 ```bash
-docker run -d -p 1935:1935 --name nginx-rtmp tiangolo/nginx-rtmp
+docker run -p 1935:1935 -p 80:80 -v hlsvol:/mnt/hls g-nginx-rtmp
```

Comment:
This isn't a publicly available container, right? How do you intend to ship this to the machine running it?

(Best see how we run the other nginx containers from the FCOS boxes)